bundles/action/org.openhab.action.ciscospark/OSGI-INF/action.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2017 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" configuration-policy="require" deactivate="deactivate" immediate="true" name="org.openhab.action.ciscospark.action">
```

Comment:
I was referring to a file like [this one](https://github.com/openhab/openhab1-addons/blob/master/bundles/binding/org.openhab.binding.garadget/OSGI-INF/binding.xml), but since your testing shows that the `service.pid` property is working with the "require" configuration policy, then I would say that no change is needed.